src/swap/components/Swap.tsx

```diff
@@ -100,9 +100,9 @@ export function Swap({ address, children, title = 'Swap' }: SwapReact) {
         return;
       }
       try {
```

Comment:
@abcrane123 follow up PR. This is getting harder to read, let's add comments to explain for each loading state what's going on.